### PR TITLE
Creator class

### PR DESCRIPTION
**Pull request by jaw772**
Create creator class and tests

### CODE REVIEW

**Review by jacksonvaldez — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -25,6 +25,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-20
+  x86_64-darwin-20
```
Comment (tjroeder) [inline Gemfile.lock:28]: Using Rosetta eh?

File: lib/creator.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    league = "this is league"
+    self.new(league)
+  end
+
+  def stat_obj_creator(game_team_data)
+    require "pry"; binding.pry
+  end
+
+
+end
```
Comment (tjroeder) [inline lib/creator.rb:18]: This looks like a good start.

**Review by tjroeder — APPROVED** [review]
Looks good. Approved.

File: Gemfile.lock (hunk 1 above)
Comment (Henchworm) [reply Gemfile.lock:28]: Rosetta for life!

**Review by Henchworm — COMMENTED** [review]
Thanks!

**Pull request merged**